adam/perception/perception_graph.py

```diff
@@ -931,11 +931,44 @@ def copy_with_temporal_scopes(
 
         return PerceptionGraphPattern(dynamic=True, graph=wrapped_graph)
 
+    def copy_remove_temporal_scopes(self) -> "PerceptionGraphPattern":
```

Comment:
Same nit as for templates:

```suggestion
    def copy_removing_temporal_scopes(self) -> "PerceptionGraphPattern":
```